server.rb

```diff
@@ -1,31 +1,31 @@
-require "sinatra"
-require "sinatra/activerecord"
+require 'sinatra'
+require 'sinatra/activerecord'
 require 'sinatra/flash'
 require 'action_mailer'
 require_relative 'app/models/user'
 require_relative 'app/mailers/user_mailer'
 require_relative 'app/mailers/application_mailer'
-set :views, Proc.new { File.join(root, "app/views") }
 
+set :views, Proc.new { File.join(root, "app/views") }
 
 configure do
   enable :sessions
   set :session_secret, "secret"
-  ActionMailer::Base.raise_delivery_errors = true
+
   ActionMailer::Base.delivery_method = :smtp
+  ActionMailer::Base.raise_delivery_errors = true
+  ActionMailer::Base.view_paths = File.expand_path('../../tps/app/views/', __FILE__)
```

Comment:
@tobara This was the offending line... I never would have noticed it in Slack, but it took me copy-pasting the line from a tutorial to realize that `Base.view_paths` needs to access the directory that CONTAINS action mailer template directories - `app/views` rather than `app/views/user_mailer`